### PR TITLE
[stable/mysql] Add service annotations using '.Values.service'

### PR DESCRIPTION
**Pull request by imcheck**
#### What this PR does / why we need it:
This pr is for selecting mysql service using annotations by external services.
Many other charts have been made like the format so that we can make it easily.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/mysql/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: mysql
-version: 0.12.0
+version: 0.12.1
```
Comment (olemarkus) [inline stable/mysql/Chart.yaml]: This adds a new feature, so could you bump the minor version?

File: stable/mysql/README.md [hunk 2]
```diff
@@ -90,6 +90,7 @@ The following table lists the configurable parameters of the MySQL chart and the
 | `metrics.readinessProbe.timeoutSeconds`      | When the probe times out                                                                     | 1                                                    |
 | `resources`                                  | CPU/Memory resource requests/limits                                                          | Memory: `256Mi`, CPU: `100m`                         |
 | `configurationFiles`                         | List of mysql configuration files                                                            | `nil`                                                |
+| `service.annotations`                        | Kubernetes annotations                                                                       | {}                                                   |
```
Comment (olemarkus) [inline stable/mysql/README.md]: Could you specify that this is annotations for the service specifically?

**Review by olemarkus — CHANGES_REQUESTED** [review]
Thanks for the contribution. Looks pretty good. Just some nitpicking.

File: stable/mysql/README.md (hunk 2 above)
Comment (imcheck) [reply stable/mysql/README.md]: Thank you for all your feedback :)

File: stable/mysql/Chart.yaml (hunk 1 above)
Comment (sameersbn) [reply stable/mysql/Chart.yaml]: ping @imcheck

**Pull request merged**